workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileEnabler.cs

```diff
@@ -6,61 +6,19 @@ public class TileEnabler : MonoBehaviour
     {
         public Transform PlayerTransform;
 
-        private bool renderersEnabled = true;
-
         public float CheckoutDistance
         {
             set => CheckoutDistanceSquared = Mathf.Pow(value, 2);
         }
 
         private float CheckoutDistanceSquared;
 
-        private MeshRenderer meshRenderer;
-
         public void Initialize(bool isClient)
         {
             if (!isClient)
             {
                 Destroy(this);
```

Comment:
So what does this script do now?

Reply:
Nothing, until Jess's stuff is merged in. Removing this now will require changes in the worker connectors and handler, so I want to avoid those merge conflicts.